.travis.yml

```diff
@@ -52,6 +52,12 @@ before_install:
   - mysql -e 'create database olympia;'
   - export GOPATH=$HOME/go
   - export PATH=$HOME/usr/local/go/bin:$GOPATH/bin:$PATH
+  # Unfortunately we require the most recent libmagic-dev version to make use of more recent
+  # JSON detection features which we require for the code-manager related APIs
+  - wget http://security.ubuntu.com/ubuntu/pool/main/f/file/libmagic-dev_5.35-4_amd64.deb
+  - wget http://security.ubuntu.com/ubuntu/pool/main/f/file/libmagic1_5.35-4_amd64.deb
+  - wget http://security.ubuntu.com/ubuntu/pool/main/f/file/libmagic-mgc_5.35-4_amd64.deb
+  - sudo dpkg -i libmagic-dev_5.35-4_amd64.deb libmagic1_5.35-4_amd64.deb libmagic-mgc_5.35-4_amd64.deb
```

Comment:
Can you figure out a way to add a repo instead like you do in the `Dockerfile` ? Really not a fan of all those hardcoded versions here.

Reply:
Probably, I'll try to make use of https://packages.ubuntu.com/disco/libmagic-dev which is at least a released (non testing) version.